+ source /root/mugen/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the test environment.'
+ message='Start to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment.'
Thu Jul 27 21:58:21 2023 - INFO  - Start to prepare the test environment.
+ EXECUTE_T=120m
+ DNF_INSTALL dmidecode
+ pkgs=dmidecode
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen/libs/locallibs/rpm_manage.py install --pkgs dmidecode --node 1 --tempfile ''
+ tmpfile2=/tmp/tmp2ef3rc2o
+ '[' -z '' ']'
+ tmpfile=/tmp/tmp2ef3rc2o
+ LOG_INFO 'End to prepare the test environment.'
+ message='End to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to prepare the test environment.'
Thu Jul 27 21:59:30 2023 - INFO  - End to prepare the test environment.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Thu Jul 27 21:59:32 2023 - INFO  - Start to run test.
+ repet_num=0
+ '[' 0 -le 50 ']'
+ echo '===========loop 0=============='
===========loop 0==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 21:59:33 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 21:59:35 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 21:59:36 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 21:59:43 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 1 -le 50 ']'
+ echo '===========loop 1=============='
===========loop 1==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 21:59:44 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 21:59:46 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 21:59:48 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 21:59:55 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 2 -le 50 ']'
+ echo '===========loop 2=============='
===========loop 2==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 21:59:56 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 21:59:58 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 21:59:59 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:00:07 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 3 -le 50 ']'
+ echo '===========loop 3=============='
===========loop 3==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:00:09 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:00:10 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:00:12 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:00:20 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 4 -le 50 ']'
+ echo '===========loop 4=============='
===========loop 4==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:00:21 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:00:23 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:00:25 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:00:32 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 5 -le 50 ']'
+ echo '===========loop 5=============='
===========loop 5==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:00:34 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:00:35 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:00:37 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:00:44 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 6 -le 50 ']'
+ echo '===========loop 6=============='
===========loop 6==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:00:45 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:00:47 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:00:49 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:00:56 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 7 -le 50 ']'
+ echo '===========loop 7=============='
===========loop 7==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:00:57 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:00:59 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:01:00 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:01:08 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 8 -le 50 ']'
+ echo '===========loop 8=============='
===========loop 8==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:01:09 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:01:11 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:01:13 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:01:20 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 9 -le 50 ']'
+ echo '===========loop 9=============='
===========loop 9==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:01:22 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:01:23 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:01:25 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:01:32 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 10 -le 50 ']'
+ echo '===========loop 10=============='
===========loop 10==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:01:34 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:01:36 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:01:37 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:01:45 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 11 -le 50 ']'
+ echo '===========loop 11=============='
===========loop 11==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:01:46 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:01:48 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:01:49 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:01:57 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 12 -le 50 ']'
+ echo '===========loop 12=============='
===========loop 12==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:01:58 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:02:00 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:02:01 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:02:09 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 13 -le 50 ']'
+ echo '===========loop 13=============='
===========loop 13==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:02:10 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:02:12 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:02:13 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:02:20 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 14 -le 50 ']'
+ echo '===========loop 14=============='
===========loop 14==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:02:21 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:02:23 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:02:24 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:02:31 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 15 -le 50 ']'
+ echo '===========loop 15=============='
===========loop 15==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:02:33 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:02:34 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:02:36 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:02:42 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 16 -le 50 ']'
+ echo '===========loop 16=============='
===========loop 16==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:02:44 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:02:45 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:02:47 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:02:53 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 17 -le 50 ']'
+ echo '===========loop 17=============='
===========loop 17==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:02:55 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:02:56 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:02:58 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:03:05 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 18 -le 50 ']'
+ echo '===========loop 18=============='
===========loop 18==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:03:06 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:03:08 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:03:09 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:03:16 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 19 -le 50 ']'
+ echo '===========loop 19=============='
===========loop 19==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:03:17 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:03:19 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:03:20 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:03:27 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 20 -le 50 ']'
+ echo '===========loop 20=============='
===========loop 20==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:03:28 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:03:30 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:03:31 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:03:38 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 21 -le 50 ']'
+ echo '===========loop 21=============='
===========loop 21==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:03:40 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:03:41 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:03:43 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:03:50 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 22 -le 50 ']'
+ echo '===========loop 22=============='
===========loop 22==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:03:51 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:03:53 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:03:54 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:04:01 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 23 -le 50 ']'
+ echo '===========loop 23=============='
===========loop 23==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:04:03 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:04:04 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:04:06 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:04:13 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 24 -le 50 ']'
+ echo '===========loop 24=============='
===========loop 24==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:04:15 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:04:16 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:04:18 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:04:25 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 25 -le 50 ']'
+ echo '===========loop 25=============='
===========loop 25==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:04:27 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:04:28 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:04:30 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:04:36 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 26 -le 50 ']'
+ echo '===========loop 26=============='
===========loop 26==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:04:38 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:04:39 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:04:41 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:04:48 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 27 -le 50 ']'
+ echo '===========loop 27=============='
===========loop 27==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:04:50 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:04:51 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:04:53 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:04:59 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 28 -le 50 ']'
+ echo '===========loop 28=============='
===========loop 28==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:05:01 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:05:02 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:05:04 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:05:11 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 29 -le 50 ']'
+ echo '===========loop 29=============='
===========loop 29==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:05:13 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:05:14 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:05:15 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:05:22 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 30 -le 50 ']'
+ echo '===========loop 30=============='
===========loop 30==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:05:23 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:05:25 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:05:26 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:05:33 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 31 -le 50 ']'
+ echo '===========loop 31=============='
===========loop 31==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:05:35 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:05:36 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:05:38 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:05:44 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 32 -le 50 ']'
+ echo '===========loop 32=============='
===========loop 32==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:05:46 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:05:47 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:05:49 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:05:56 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 33 -le 50 ']'
+ echo '===========loop 33=============='
===========loop 33==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:05:57 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:05:58 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:06:00 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:06:07 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 34 -le 50 ']'
+ echo '===========loop 34=============='
===========loop 34==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:06:09 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:06:10 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:06:12 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:06:18 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 35 -le 50 ']'
+ echo '===========loop 35=============='
===========loop 35==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:06:20 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:06:22 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:06:23 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:06:30 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 36 -le 50 ']'
+ echo '===========loop 36=============='
===========loop 36==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:06:32 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:06:33 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:06:35 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:06:42 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 37 -le 50 ']'
+ echo '===========loop 37=============='
===========loop 37==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:06:44 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:06:45 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:06:47 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:06:54 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 38 -le 50 ']'
+ echo '===========loop 38=============='
===========loop 38==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:06:55 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:06:57 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:06:59 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:07:06 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 39 -le 50 ']'
+ echo '===========loop 39=============='
===========loop 39==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:07:08 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:07:09 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:07:11 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:07:18 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 40 -le 50 ']'
+ echo '===========loop 40=============='
===========loop 40==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:07:19 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:07:21 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:07:22 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:07:29 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 41 -le 50 ']'
+ echo '===========loop 41=============='
===========loop 41==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:07:31 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:07:32 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:07:34 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:07:40 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 42 -le 50 ']'
+ echo '===========loop 42=============='
===========loop 42==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:07:42 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:07:43 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:07:45 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:07:52 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 43 -le 50 ']'
+ echo '===========loop 43=============='
===========loop 43==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:07:53 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:07:55 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:07:56 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:08:03 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 44 -le 50 ']'
+ echo '===========loop 44=============='
===========loop 44==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:08:05 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:08:06 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:08:08 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:08:14 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 45 -le 50 ']'
+ echo '===========loop 45=============='
===========loop 45==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:08:16 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:08:17 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:08:19 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:08:25 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 46 -le 50 ']'
+ echo '===========loop 46=============='
===========loop 46==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:08:27 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:08:28 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:08:30 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:08:37 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 47 -le 50 ']'
+ echo '===========loop 47=============='
===========loop 47==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:08:39 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:08:40 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:08:41 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:08:48 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 48 -le 50 ']'
+ echo '===========loop 48=============='
===========loop 48==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:08:50 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:08:51 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:08:53 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:09:00 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 49 -le 50 ']'
+ echo '===========loop 49=============='
===========loop 49==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:09:01 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:09:03 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:09:04 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:09:11 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 50 -le 50 ']'
+ echo '===========loop 50=============='
===========loop 50==============
+ SSH_CMD ls '' '' ''
+ cmd=ls
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c ls -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:09:13 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 35'
+ message='oe_test_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 35'
Thu Jul 27 22:09:14 2023 - ERROR - oe_test_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_CMD reboot '' '' ''
+ cmd=reboot
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 22:09:16 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ ping -c 10
ping: usage error: Destination address required
+ grep -w '0% packet loss' lossfile
+ CHECK_RESULT 1 1
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ REMOTE_REBOOT_WAIT 2 10
+ node=2
+ waittime=10
+ '[' 10x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 10
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_reboot.sh line 44'
+ message='oe_test_reboot.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_reboot.sh line 44'
Thu Jul 27 22:09:22 2023 - ERROR - oe_test_reboot.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ (( repet_num++ ))
+ '[' 51 -le 50 ']'
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Thu Jul 27 22:09:23 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 102
+ test 102 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Thu Jul 27 22:09:25 2023 - ERROR - failed to execute the case.
+ exit 102
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Thu Jul 27 22:09:26 2023 - INFO  - Start to restore the test environment.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmp2ef3rc2o ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmp2ef3rc2o
+ '[' 0 -ne 0 ']'
+ rm -rf lossfile
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Thu Jul 27 22:09:47 2023 - INFO  - End to restore the test environment.
